tests/feature/test_scenarios.py

```diff
@@ -0,0 +1,59 @@
+"""Test scenarios shortcut."""
+import textwrap
+
+
+def test_scenarios(testdir):
+    """Test scenarios shortcut."""
+    testdir.makeconftest("""
+        import pytest
+        from pytest_bdd import given
+
+        @given('I have a bar')
+        def i_have_bar():
+            print('bar!')
```

Comment:
i missed the assert of the printed output - just forgot

On 18 March 2015 at 16:07, Daniel Hahler <notifications@github.com> wrote:

> In tests/feature/test_scenarios.py
> <https://github.com/pytest-dev/pytest-bdd/pull/106#discussion_r26671227>:
>
> > @@ -0,0 +1,59 @@
> > +"""Test scenarios shortcut."""
> > +import textwrap
> > +
> > +
> > +def test_scenarios(testdir):
> > +    """Test scenarios shortcut."""
> > +    testdir.makeconftest("""
> > +        import pytest
> > +        from pytest_bdd import given
> > +
> > +        @given('I have a bar')
> > +        def i_have_bar():
> > +            print('bar!')
>
> This seems to be done via the outer test (looking at the output).
> Or is there something specific to assert in the inner test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pytest-dev/pytest-bdd/pull/106/files#r26671227>.
>



-- 
Anatoly Bubenkov